cssgunc/CSPortal
# Generate routes

## Summary 
Generated links for each individual webinar and opportunity when clicked, based on the id. Added routes to App -> index.js to deal with connecting that link to the proper subpage component. Also added corresponding variable names to constants -> routes.js for the individual opportunity/webinar links.

## To test 
1. Check out the sample-pull-request branch `git checkout generate-routes`
2. Run `npm start` in your command line inside the RTCPortal root directory
3. Navigate to the Opportunities page: Resources > Opportunities
4. Click through the individual opportunities to see the links generated in the address bar.
5. Repeat the same process for Webinars.

## Test cases 
<img width="1440" alt="Screen Shot 2020-10-20 at 9 43 39 PM" src="https://user-images.githubusercontent.com/43039779/96662862-61c1c400-131d-11eb-9b72-9458d05582e6.png">
<img width="1440" alt="Screen Shot 2020-10-20 at 9 43 41 PM" src="https://user-images.githubusercontent.com/43039779/96662867-65554b00-131d-11eb-84ca-b341ae177e71.png">
<img width="1440" alt="Screen Shot 2020-10-20 at 9 43 45 PM" src="https://user-images.githubusercontent.com/43039779/96662871-67b7a500-131d-11eb-89f5-bb85acc1914b.png">
<img width="1440" alt="Screen Shot 2020-10-20 at 9 43 48 PM" src="https://user-images.githubusercontent.com/43039779/96662874-6a19ff00-131d-11eb-9a9a-c0592d7b3535.png">
<img width="1440" alt="Screen Shot 2020-10-20 at 9 43 52 PM" src="https://user-images.githubusercontent.com/43039779/96662880-6be3c280-131d-11eb-8547-306afd60e7c8.png">
